### PR TITLE
WIP: carry over the quote character to output same quote in html

### PR DESCRIPTION
**Pull request by tardyp**
this is WIP, as lots of unit test must be updated to reflect original quote.

upload first to get feedback before doing the painfull test update

Fix: #60

### CODE REVIEW

File: pypugjs/testsuite/cases/text.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<option value="">-- (selected) --</option>
```
Comment (tardyp) [inline pypugjs/testsuite/cases/text.html:1]: we'll have to do similar patch for lots of test fixtures (as original pug quote is ' ). I want to know if this is fine first

Comment (kakulukia) [reply pypugjs/testsuite/cases/text.html:1]: well, this doesnt sound right .. the quote char, whatever it is, shall be untouched and thus not breaking existing tests or are the tests putting ' in and expect " to come out?

I can probably could have a deeper look tonight.

Comment (tardyp) [reply pypugjs/testsuite/cases/text.html:1]: > are the tests putting ' in and expect " to come out?

Right. I think the test are incorrect due to the original behaviour was incorrect.
I would expect pypugjs should behave as much as possible the same as pugjs.

Even if probably this behaviour of keeping the quote char is not "specified".

I can try to run the test suite against pugjs to prove that point.

**Comment by tardyp** [comment]
So I did the check with pugjs it actually quote the quotes (as originally reported 🤦) 

```
+<div ng-hide="status == &quot;Live&quot;"></div>
```

So my patch does not make sense

**Pull request closed without merging**